packages/valory/connections/p2p_libp2p/consts.py

```diff
@@ -22,6 +22,20 @@
 import platform
 
 
+PUBLIC_STAGING_DHT_P2P_MADDR_1 = "/dns4/acn.staging.autonolas.tech/tcp/9003/p2p/16Uiu2HAm9ftkcmsBwPf2KXjrUd9G6GPi2WN3opXvjrjukNpE9e5k"
```

Comment:
Why on the package level? This was more appropriate in the tests imo. IT is not generic enough for the package

Reply:
well I'll change it now. This is how it is defined in the connection.yaml file and ideally this would be referenced in there as some sort of environmental variable.